includes/Admin/Cross_Origin_Isolation.php

```diff
@@ -118,40 +101,36 @@ public function admin_footer() {
 	 *
 	 * @param string $html HTML document as string.
 	 *
-	 * @return string
+	 * @return string Processed HTML document.
 	 */
 	protected function replace_in_dom( string $html ): string {
-		$document = Document::fromHtml( $html );
-
-		if ( ! $document ) {
-			return $html;
-		}
-
-		$map = [
-			'link'   => 'href',
-			'img'    => 'src',
-			'iframe' => 'src',
-			'script' => 'src',
+		$tags = [
+			'link',
```

Comment:
I am wonder if we should also do video and audio tags as well here?

Reply:
Yeah makes sense.

And actually need to remove `iframe` because `crossorigin` is not a valid attribute on iframes.